charts/kof-child/templates/child-multi-cluster-service.yaml

```
@@ -48,26 +48,39 @@ opentelemetry-kube-stack:
     {{- $customRegistry = $global.registry }}
   {{- end }}
 {{- end -}}
+
+{{ range $istio := list true false }}
+---
 apiVersion: k0rdent.mirantis.com/v1beta1
 kind: MultiClusterService
 metadata:
+{{ if $istio }}
+  name: kof-istio-child-cluster
+{{ else }}
   name: kof-child-cluster
+{{ end }}
+
```

Sorry, I've forgot about `{{-`
```suggestion
{{- if $istio }}
   name: kof-istio-child-cluster
{{- else }}
  name: kof-child-cluster
{{- end }}
```